src/ui/header/LinkList.tsx

```
@@ -0,0 +1,22 @@
+
+export default function LinkList({ link, links }: Sanity.LinkList) {
+  return (
+    <InteractiveDetails className="group relative" name="header" delay={10} closeAfterNavigate>
+      <summary className="flex h-full items-center gap-1 text-center md:px-3 md:leading-tight">
+        {link?.label}
+        <CgChevronRight className="shrink-0 transition-transform group-open:rotate-90 md:rotate-90" />
+      </summary>
+
+      <ul className="anim-fade-to-b md:bg-background/95 border-foreground/10 top-full left-0 px-3 py-2 max-md:border-l md:absolute md:min-w-max md:rounded md:border md:shadow-md">
+        {links?.map((link, key) => (
+          <li key={key}>
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)